src/fsharp/DependencyManager.Integration.fs

```diff
@@ -0,0 +1,236 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+/// Helper members to integrate DependencyManagers into F# codebase
+module internal Microsoft.FSharp.Compiler.DependencyManagerIntegration
+
+open System
+open System.Reflection
+open System.IO
+open Microsoft.FSharp.Compiler.ErrorLogger
+
+// NOTE: this contains mostly members whose intents are :
+// * to keep ReferenceLoading.PaketHandler usable outside of F# (so it can be used in scriptcs & others)
+// * to minimize footprint of integration in fsi/CompileOps
+
+/// hardcoded to net461 as we don't have fsi on netcore
+let targetFramework = "net461"
+
+module ReflectionHelper =
+    let assemblyHasAttribute (theAssembly: Assembly) attributeName =
+        try
+            CustomAttributeExtensions.GetCustomAttributes(theAssembly)
+            |> Seq.tryFind (fun a -> a.GetType().Name = attributeName)
+            |> function | Some _ -> true | _ -> false
+        with | _ -> false
+
+    let getAttributeNamed (theType: Type) attributeName =
+        try
+            theType.GetTypeInfo().GetCustomAttributes false
+            |> Seq.tryFind (fun a -> a.GetType().Name = attributeName)
+        with | _ -> None
+
+    let getInstanceProperty<'treturn> (theType: Type) indexParameterTypes propertyName =
```

Comment:
I believe indexParameterTypes is always empty in this PR, I think it's best to remove it.